### ISSUE TITLE
[Feature request] Rendering an initial count of arrays (listfields)

### ISSUE THREAD

**Issue by serkandurusoy** [vazco/uniforms#14]
TL:DR;
add an "initialCount" property for listfields

--- 

Autoform has a great feature with handling array structures where

* if there are `minCount` and `maxCount` defined on the schema
 * autoform displays minimum of minCount children ready to be filled and no [-] button
 * allows maximum of maxCount children and does not display [+] button after the last child
* if there is an `initialCount` defined as a property of the [Array] field
 * displays initialCount children in the first render (if it is not less than minCount)
* if the array is *not* `optional: true` (meaning it should contain at least one child)
 * always displays at least one child

This allows for better UX design for forms with nested structures.

Uniforms does take into account minCount and maxCount but it lacks an "initialCount" feature.

Therefore, uniform's handling of such arrays of fields requires user interaction to be able to render the initial child input and users get confused about it when they don't see that first child input there and only a [+] button

**Comment by serkandurusoy** [comment]
Thank you!

**Comment by serkandurusoy** [comment]
@radekmie could you push a release to npm?

**Comment by radekmie** [comment]
I've published `1.0.0-rc.8`.

**Comment by radekmie** [comment]
It's fixed in `1.0.0-rc.9`.

**Comment by serkandurusoy** [comment]
Thanks!
